pkgs/tools/misc/octosql/default.nix

```
@@ -0,0 +1,33 @@
+{ lib
+, buildGo118Module
+, fetchFromGitHub
+}:
+
+with lib;
+
+buildGo118Module rec {
```

nixpkgs uses buildGo118Module by default
```suggestion
, buildGoModule
, fetchFromGitHub
}:

buildGoModule rec {
```

---

Thank you for pointing out!